parser.go

```diff
@@ -682,7 +681,7 @@ func protectedParser(l *State, r io.Reader, name, chunkMode string) error {
 		} else if c == Signature[0] {
 			l.checkMode(chunkMode, "binary")
 			b.UnreadByte()
-			closure, err = l.undump(b, name) // TODO handle err
+			closure, _ = l.undump(b, name) // TODO handle err
```

Comment:
Are we planning on handling the error here? If not, can we remove the TODO comment?

Reply:
This was simply to remove a warning that was caused by an unused variable. The TODO still stands and if we handle it, I would do this in another PR.